packages/design/src/DataTableNext/Pager/Pager.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Flex, Text } from 'design';
+import { CircleArrowLeft, CircleArrowRight } from 'design/Icon';
+import { StyledArrowBtn, StyledFetchMoreBtn } from './StyledPager';
+import usePager, { State, Props } from './usePager';
+
+export default function Container(props: Props) {
+  const state = usePager(props);
+  return <Pager {...state} />;
+}
+
+export function Pager({
+  nextPage,
+  prevPage,
+  isNextDisabled,
+  isPrevDisabled,
+  from,
+  to,
+  count,
+  fetchMore,
+}: State) {
+  return (
+    <Flex justifyContent={fetchMore ? 'space-between' : 'end'} width="100%">
+      {fetchMore && (
+        <StyledFetchMoreBtn onClick={fetchMore}>Fetch More</StyledFetchMoreBtn>
+      )}
```

Comment:
Maybe i missed it, but I'm not sure i see how fetch states are handled, if you look at our old `Pager.jsx`, you'll see `fetchStatus` which handles cases like:

- nothing more to fetch (disabled)
- fetch loading (loading), which disables this button
- can fetch (blank state)